.github/workflows/build_test_images.yaml

```diff
@@ -68,6 +92,8 @@ jobs:
           ENVIRONMENT: ${{ vars.TARGET_CLOUD }}
           PACKER_TEMPLATE: ${{ matrix.template }}
           ENV_VAR_FILES: ${{ matrix.var-files }}
+          PKR_VAR_source_image_name: ${{ needs.ensure_source_image.outputs.source_image }}
+          SKIP_SOURCE_IMAGE: "true"
```

Comment:
Should we be explicitly checking whether the image already exists on the target cloud as part of the `ensure-source-image` script instead of always skipping? Currently I think this means we'd need to manually upload the source image any time we update to a newer source image or want to run CI on a new cloud.